scalameter-core/src/test/scala/org/scalameter/picklers/PicklersSpecification.scala

```diff
@@ -0,0 +1,89 @@
+package org.scalameter.picklers
+
+import java.util.Date
+import org.scalameter.picklers.Implicits._
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{FunSuite, Matchers}
+
+
+class PicklerSpecification extends FunSuite with PropertyChecks with Matchers {
+  def validatePickler[T: Pickler](o: T) = implicitly[Pickler[T]].unpickle(implicitly[Pickler[T]].pickle(o)) should === (o)
+
+  test("Byte pickling") {
+    forAll { o: Byte =>
```

Comment:
Btw - is it possible to configure the test with ScalaTest's `PropertyChecks`?
E.g. number of tests, seed for the random value generator, minimization?